src/systemPrompt.ts

```diff
@@ -1,4 +1,4 @@
-import os from "os";
+import os from "node:os";
```

Comment:
bun aims to have full compatibility with node, so I don't see the need in the prefix

Reply:
I'm super confused why you think that having compatibility with node APIs would imply not using proper namespaces...

Reply:
https://x.com/jarredsumner/status/1811521754109800609?s=46

Reply:
Yes, they're optional. It's still ***best practice***.